### PR TITLE
Version 1.9.1

### PR DESCRIPTION
**Pull request by SC426**
fastavro 1.9.1

**Destination channel:** defaults

### Links

- [PKG-3332](https://anaconda.atlassian.net/browse/PKG-3332)
- [Upstream repository](https://github.com/fastavro/fastavro/tree/1.9.1)
- [Upstream changelog/diff](https://github.com/fastavro/fastavro/blob/1.9.1/ChangeLog)

### Explanation of changes:

- Bump version to `1.9.1`
- Update Python skip to `<38`

[PKG-3332]: https://anaconda.atlassian.net/browse/PKG-3332?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-  sha256: 4b1205f46489b4032d3155c1ab44d9824be0c7454df98d3a5bd22b78b98f23c8
+  sha256: f37011d66de8ba81b26760db0478009a14c08ebfd34269b3390abfd4616b308f
 
 build:
   number: 0
-  skip: True  # [py<37]
+  skip: True  # [py<38]
   script:
     - export FASTAVRO_USE_CYTHON=1  # [not win]
     - set FASTAVRO_USE_CYTHON=1  # [win]
```
Comment (danpetry) [inline recipe/meta.yaml:17]: could you add --no-build-isolation to pip install

**Review by danpetry — APPROVED** [review]
LGTM apart from one comment

**Review by psteyer — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:17]: I think setting `FASTAVRO_USE_CYTHON` is not needed anymore.

Comment (SC426) [reply recipe/meta.yaml:17]: This PR is already merged. Should I open another to change this?

Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml:17]: You can, but it's not mandatory. My comment was more informational than anything else. Setting `FASTAVRO_USE_CYTHON` to `1` doesn't have any effect as far as I see, so you can leave this to the next person that will update this package.

**Pull request merged**